openresty/nginx/conf/includes/root_location.conf

```diff
@@ -27,10 +27,11 @@ location / {
   }
 
   if ($request_method = 'DELETE') {
-    add_header 'Access-Control-Max-Age' 1728000;
-    add_header 'Content-Type' 'text/plain charset=UTF-8';
-    add_header 'Content-Length' 0;
-    return 405;
+      set_by_lua_block $dummy {
+          ngx.header['Access-Control-Allow-Methods'] = 'GET, POST, PATCH, DELETE, OPTIONS';
+          ngx.header['Access-Control-Allow-Credentials'] = 'true';
+          return true;
+      }
```

Comment:
@ciaranevans How does this work to prevent full table deletes?  I was imagining that we would need to add some logic to assert that an `id` was present in the query params in this area.

Reply:
Good question, I'm not sure. It might come down to this having to be a client check... I'll do some reading into it thought for sure